pywbem_mock/_wbemconnection_mock.py

```diff
@@ -497,6 +500,67 @@ def install_namespace_provider(self, interop_namespace,
                                schema_pragma_files=schema_pragma_file,
                                verbose=verbose)
 
+    def install_subscription_providers(self, interop_namespace,
+                                       schema_pragma_file=None,
+                                       verbose=None):
+        """
```

Comment:
Mark it experimental, since we think it will go away again.

Reply:
Marked as experimental inn the code